checkov/terraform/checks/provider/linode/__init__.py

```diff
@@ -1,5 +1,4 @@
-from os.path import dirname, basename, isfile, join
-import glob
+from pathlib import Path
 
-modules = glob.glob(join(dirname(__file__), "*.py"))
-__all__ = [basename(f)[:-3] for f in modules if isfile(f) and not f.endswith("__init__.py")]
+modules = Path(__file__).parent.glob("*.py")
```

Comment:
what does this change?

Reply:
the functionality is the same, it just uses the "newer" builtin module `pathlib` which is object oriented and supports Windows file paths much better.